third_party/terraform/utils/utils.go.erb

```diff
@@ -423,3 +423,26 @@ func stringInSlice(arr []string, str string) bool {
 func migrateStateNoop(v int, is *terraform.InstanceState, meta interface{}) (*terraform.InstanceState, error) {
 		return is, nil
 }
+
+func expandString(v interface{}, d TerraformResourceData, config *Config) (string, error) {
+	return v.(string), nil
+}
+
+func expandSourceRepoRepositoryPubsubConfigsTopic(v interface{}, d TerraformResourceData, config *Config) (string, error) {
```

Comment:
Can you move this to constants for pubsub or pubsub_utils.go instead of the utils.go?

Reply:
Added as source_repo_utils (since this is a subfield on SourceRepo Repo that configures pubsub)

Reply:
Makes sense, but you'll need to add that file to the terraform-conversion mapper or the downstream will fail to build.